src/client/datascience/raw-kernel/rawKernel.ts

```diff
@@ -111,7 +113,16 @@ export class RawKernel implements Kernel.IKernel {
         await this.kernelProcess.dispose();
         this.socket.dispose();
     }
-    public getSpec(): Promise<Kernel.ISpecModel> {
+    public async getSpec(): Promise<Kernel.ISpecModel> {
+        if (this.kernelProcess.kernelConnectionMetadata.kind === 'startUsingKernelSpec') {
+            const kernelSpec = cloneDeep(this.kernelProcess.kernelConnectionMetadata.kernelSpec) as any;
+            const resources = 'resources' in kernelSpec ? kernelSpec.resources : {};
```

Comment:
When using raw kernels, we have the kernel spec, hence return that.
Not used today, but will be used in long running kernels, noop change.